super_project/CMakeLists.txt

```diff
@@ -16,17 +12,34 @@ else()
 endif()
 
 
-list(APPEND CMAKE_MODULE_PATH
-    ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-
+include(FetchContent)
+FetchContent_Declare(
+  farm_ng_cmake
+  GIT_REPOSITORY https://github.com/farm-ng/farm-ng-cmake.git
+  GIT_TAG main
```

Comment:
Better use a version tag or sha here.

Strictly speaking this is leading to non-deterministic builds. Probably not too big of a deal in everyday development, since farm-ng-cmake repo should not change to frequently.

However, if somebody were to depend on this super-project in their downstream library / project, this could magically break them overnight - and the cause might be quite hard to find.

Reply:
Yes, that's true, but a tag or branch may be easier to track then a sha.  We can hard code the sha in there now if you like.

Reply:
Ok for farm-ng-core i added a sha.